src/test/unit/comparison.test.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import { getEmptyProfile } from '../../profile-logic/data-structures';
+import { mergeProfiles } from '../../profile-logic/comparison';
+
+describe('mergeProfiles function', function() {
+  it('should set interval of merged profile to minimum of all intervals', function() {
+    const profile1 = getEmptyProfile();
+    const profile2 = getEmptyProfile();
```

Comment:
I think we want profiles that actually have something inside (in this case we have 2 empty profiles, this means profiles with no threads -> we can't merge anything, the function is a bit pointless)

You can use the util `getProfileFromTextSamples` we use in a lot of other places. For example: `getProfileFromTextSamples('A')` will get you a simple profile. You can do that twice, maybe with another letter for more variety, and you'll have 2 interesting profiles.